### PR TITLE
Bug 2090730: Adds collection of Multus log files

### PR DESCRIPTION
**Pull request by nicklesimba**
Script added that gathers multus logs from pods across nodes

Signed-off-by: nicklesimba <simha.nikhil@gmail.com>

### CODE REVIEW

File: collection-scripts/gather_network_logs [hunk 1]
```diff
@@ -15,6 +15,20 @@ function gather_multus_data {
...
+INPUT_LOG_PATH="/var/run/multus/cni/net.d/multus.log"
+OUTPUT_LOG_PATH=${OUT:-"${NETWORK_LOG_PATH}/multus_logs"}
+
+mkdir -p "${OUTPUT_LOG_PATH}"/
+
+function gather_multus_logs {
+  for DAEMON_POD in ${MULTUS_DAEMON_PODS[@]}; do
+  # Collect var/log/multus.log if exists.
+    out=$(oc exec -n openshift-multus "${DAEMON_POD}" -- \
+    /bin/bash -c "cat $INPUT_LOG_PATH" 2>/dev/null) && echo "$out" 1> "${OUTPUT_LOG_PATH}"
```
Comment (deads2k) [inline collection-scripts/gather_network_logs:28]: if this file isn't present do we continue? We need to

File: collection-scripts/gather_network_logs [hunk 2]
```diff
@@ -15,6 +15,20 @@ function gather_multus_data {
   oc get multi-networkpolicy -o yaml --all-namespaces > "${NETWORK_LOG_PATH}"/multi_networkpolicy 2>&1 & PIDS+=($!)
 }
 
+MULTUS_DAEMON_PODS="${@:-$(oc -n openshift-multus get pods -l app=multus -o jsonpath='{.items[*].metadata.name}')}"
```
Comment (deads2k) [inline collection-scripts/gather_network_logs:18]: does this run by default?  if so, this must be restricted to masters.

Comment (dougbtv) [reply collection-scripts/gather_network_logs:18]: For the purposes of debugging the sandbox creation failures -- wouldn't we want the logs from all nodes?

**Comment by dougbtv** [comment]
/retitle Bug 2090730: Adds collection of Multus log files

**Pull request merged**